packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -11,7 +11,7 @@ set -u
 # SCRIPT KNOBS
 #######################################################################
 # Update for new releases, will pull this tag in the repo
-DEFAULT_AGENT_VERSION="5.14.1"
+DEFAULT_AGENT_VERSION="5.16.0"
```

Comment:
if it upgrade to 5.17 why it says everywhere 5.16?

Reply:
Good point, I don't know why they didn't update this version....
I think we should change it to 5.17.2, which seems a better default for us.